scaleway/loadbalancers_annotations.go

```diff
@@ -543,6 +553,58 @@ func getForwardProtocol(service *v1.Service, nodePort int32) (scwlb.Protocol, er
 	return scwlb.ProtocolTCP, nil
 }
 
+func getSSLBridging(service *v1.Service, nodePort int32) (*bool, error) {
+	tlsEnabled, found := service.Annotations[serviceAnnotationLoadBalancerHTTPBackendTLS]
+	if !found {
+		return nil, nil
```

Comment:
just wondering, this return just does nothing ?

Reply:
same pattern as the other, if no annotation set, we return the default (unset)

Reply:
but agreed that passing an empty string to `isPortInRange` would render the same result.
I'll keep the check as it's more explicit